install/running_install.adoc

```diff
@@ -406,19 +408,20 @@ endif::[]
 The main installation playbook *_{pb-prefix}playbooks/deploy_cluster.yml_* runs a
 set of individual component playbooks in a specific order, and the installer
 reports back at the end what phases you have gone through. If the installation
-fails during a phase, you are notified on the screen along with the errors from
+fails, you are notified which phase failed along with the errors from
 the Ansible run.
 
-After you resolve the issue, rather than run the entire installation over again,
-you can pick up from the failed phase. You must then run each of the remaining
-playbooks in order:
+After you resolve the errors, you can either rerun the *_deploy_cluster.yml_* 
```

Comment:
"you can either rerun the ... ", 
As Scott says, If it is deployed in a new environment, that is ok to re-run the thedeploy_cluster.yml
But if the cluster is already up and it has workload, It is better to  resolve the current errors, then run each of the remaining playbooks in order.

Reply:
@shlao, if you had current workloads, how often would you be installing instead of upgrading?

Reply:
I don't see the scenario: "it has workload",  @sdodson  Maybe meet this before. 
But add the the prerequisite: “deployed in a new environment“ to "re-run deploy_cluster.yml" is correctly.

Reply:
What's the difference between "a new environment" and "an environment without workloads?"

Reply:
"environment without workload“ it means that there exist some idle programs to wait the request.
"a new environment" it is ,common, means that there is nothing except the basic process of OS.

Reply:
"environment without workload", for example, the Bank System, in early morning, e.g 2:00 AM
the system without workload, no transactions from the clients.
Beside, the means of the word "environment", include: HW, Software, network and so on.

Reply:
Thanks @shlao! Is this better?